backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/Schema.kt

```diff
@@ -0,0 +1,21 @@
+package app.ehrenamtskarte.backend.application.database
+
+import app.ehrenamtskarte.backend.regions.database.Regions
+import org.jetbrains.exposed.dao.IntEntity
+import org.jetbrains.exposed.dao.IntEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IntIdTable
+
+object EakApplications : IntIdTable() {
+    val regiondId = reference("regionId", Regions)
+    val valueBlob = blob("value")
```

Comment:
blobably we dont need a blob, but just a simple string?

Reply:
According to [this ](https://engineering.pivotal.io/post/bytea_versus_text_in_postgresql/) blobs have a better performance in postgres. You can not search on them. But with structured JSON stored inside valueBlob this would be hopelessly anyway.

Also i appreciated your wordplay :P

Reply:
From your reference:
> Switching from ByteA to TEXT will result in an instant performance improvement for read queries. That is a big gain, given that no application code has to be changed. The write performance will not change much.

So I think TEXT is truly the way to go here :)